File: IDTO-master/IDTO MDT/IdtoWsComms/src/org/battelle/idto/ws/otp/data/StopTime.java

package org.battelle.idto.ws.otp.data;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.List;

public class StopTime{
   	private String phase;
   	private Number time;
   	private RouteTrip trip;

 	public String getPhase(){
		return this.phase;
	}
	public void setPhase(String phase){
		this.phase = phase;
	}
 	public Number getTime(){
		return this.time;
	}
	public void setTime(Number time){
		this.time = time;
	}
 	public RouteTrip getTrip(){
		return this.trip;
	}
	public void setTrip(RouteTrip trip){
		this.trip = trip;
	}
	
	@Override
	public String toString() {
		Date date = new Date(getTime().longValue()*1000l);
		
		SimpleDateFormat formatter = new SimpleDateFormat("hh:mm:ss a");
		return formatter.format(date) + " - " + getTrip().getTripHeadsign();
	}
}

File: IDTO-master/IDTO Azure Hosted Systems/IDTO.RouteAggregationLibrary/OpenTripPlanner/Model/StopTime.cs
namespace IDTO.RouteAggregationLibrary.OpenTripPlanner.Model
{
    public class StopTime
    {
        public int time { get; set; }
        public string phase { get; set; }
        public Trip trip { get; set; }
    }

    //public class StopTime2
    //{
    //    public StopTime StopTime { get; set; }

    //    public StopTime2()
    //    {
    //        StopTime = new StopTime {trip = new Trip()};
    //    }
    //}

}